src/hooks/screen/useChatScreenHook.ts

```diff
@@ -156,7 +179,8 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
           }
         }
 
-        refresh('chat');
+        refreshWithId('chat', selectedChannel?.id);
+        // refresh('chat');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `refreshWithId` function is used instead of the `refresh` function. This seems like a more efficient way to refresh data since it only targets data with a specific ID rather than refreshing all data. However, the old `refresh` call is commented out instead of being removed. It's generally a good practice to remove unused code to keep the codebase clean and maintainable.

```diff
        refreshWithId('chat', selectedChannel?.id);
-183:         // refresh('chat');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->